Survey/Survey/Sources/Presentation/Modules/Splash/SplashView.swift

```diff
@@ -9,10 +9,26 @@ import SwiftUI
 
 struct SplashView: View {
 
+    @State private var logoOpacity = 0.0
+
     var body: some View {
         ZStack {
-            Color(.systemTeal)
-            Text("Survey App")
+            Image(Images.splashBackground)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .edgesIgnoringSafeArea(.top)
+            Image(Images.overlay)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .edgesIgnoringSafeArea(.top)
+            Image(Images.logoWhite)
+                .resizable()
+                .frame(width: 202.0, height: 48.0, alignment: .center)
+                .opacity(logoOpacity)
+        }.onAppear {
+            withAnimation(.easeInOut(duration: 2.0)) {
+                logoOpacity = 1.0
+            }
```

Comment:
From the Figma prototype, I think the `logo` does not animate immediately on appear. It does so with a little delay I believe. So you might want to use the `delay()` modifier here 🙏

Reply:
updated in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/38/commits/ed946222cb171779b44bd26dcc147c0fbc9ed649)